### PR TITLE
Adds new integration [Sese-Schneider/ha-cover-time-based]

### PR DESCRIPTION
**Pull request by Sese-Schneider**
## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: https://github.com/Sese-Schneider/ha-cover-time-based/releases/tag/v2.0.0
Link to successful HACS action (without the `ignore` key): https://github.com/Sese-Schneider/ha-cover-time-based/actions/runs/6430678535
Link to successful hassfest action (if integration): https://github.com/Sese-Schneider/ha-cover-time-based/actions/runs/6425890011

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -806,6 +806,7 @@
   "Sennevds/media_player.template",
   "sergeylysov/sst_cloud",
   "sermayoral/ha-samsungtv-encrypted",
+  "Sese-Schneider/ha-cover-time-based",
```
[the remark this answers is not in the document]
Comment (Sese-Schneider) [reply integration:818]: I tried to figure out what you exactly mean by "not compatible", but didn't understand the issue.
Please tell me what I need to do to make it compatible @ludeeus

Things I've done:
- [x] The changed LOC is at the correct place
- [x] hassfest workflow passes [link](https://github.com/Sese-Schneider/ha-cover-time-based/actions/runs/7419623996)
- [x] HACS workflow passes [link](https://github.com/Sese-Schneider/ha-cover-time-based/actions/runs/7419624877)
- [x] Integration works in production with 2024.01 [screenshot](https://s.sese.dev/yA7E4J4pwd32k7h29)
- [x] I've rechecked the [publishing documentation](https://hacs.xyz/docs/publish/start) if I did everything here
- [x] I've added a brand [link](https://github.com/home-assistant/brands/pull/4741)

[the remark this answers is not in the document]
Comment (ludeeus) [reply integration:818]: Your current release (v1.0.0) contains invalid data in hacs.json <https://github.com/Sese-Schneider/ha-cover-time-based/blob/v1.0.0/hacs.json#L5>

[the remark this answers is not in the document]
Comment (Sese-Schneider) [reply integration:818]: @ludeeus thank you for your assistance!

The issue was that for an unknown reason the release `v1.0.0` was still marked as the latest release in the GitHub API, while the GitHub UI was correctly updated with the release `v2.0.0`.
I don't know why that has happened, might have been an issue with GitHub, but I can't tell ...
Anyways, this lead me to believe everything was fine (the tests in my repo did pass after all and `v2.0.0` was available) while in this repo the wrong version was fetched and the tests failed.

I've _republished_ the release `v2.0.0` and pushed a new one `v2.0.1` for good measure, which is the latest release now.
Everything should work now 🤞🏼 [HACS](https://github.com/Sese-Schneider/ha-cover-time-based/actions/runs/7420610132) [hassfest](https://github.com/Sese-Schneider/ha-cover-time-based/actions/runs/7420610847)

I apologize for the inconvenience

**Comment by Sese-Schneider** [comment]
Sorry about re-requesting the review, I forgot this is not allowed here and I'm used to pressing the button from work ...

**Pull request merged**